docs/posts/2025/08/Interactive-Shell.md

```diff
@@ -0,0 +1,59 @@
+---
+authors:
+  - branfosj
+date: 2025-08-09
+slug: interactive-shell
+hide:
+  - navigation
+---
+
+# Interactive Shell to Debug Failed Builds
+
+One of the big enhancements in EasyBuild v5 is the addition of an [interactive shell to debug failed builds](https://docs.easybuild.io/interactive-debugging-failing-shell-commands/).
+
+We would all prefer that no build fails, but there will be times when they do.
+The new interactive shell feature provides a way to explore the build environment and debug the failure.
+
+<!-- more -->
+
+---
+
+## Demo
+
+In the following demo, `bzip2-1.0.8.eb` is being built on an Ubuntu 25.04 (Plucky Puffin) system.
+This build uses the `SYSTEM` toolchain, which means that the OS default compiler is used - in this case `gcc` v13.3.0.
+The failure is because of an incompatibility between the older `binutils` (v2.37, as defined in the easyconfig) and the newer `gcc`.
```

Comment:
```suggestion
The compilation failure is because of an incompatibility between the older `binutils` (v2.37, as defined in the easyconfig) and the newer GCC.
```